generators/Generators.fs

```diff
@@ -147,7 +162,7 @@ type KindergartenGarden() =
 type LargestSeriesProduct() =
     inherit Exercise()
 
-     override this.PropertiesWithIdentifier canonicalDataCase = ["digits"]
+    override this.PropertiesWithIdentifier canonicalDataCase = ["digits"]
```

Comment:
See comment above

Reply:
Will change to use this.Properties canonicalDataCase.

Reply:
@vrnithinkumar Ah, I linked to the wrong property, as `this.Properties` also includes the expected value, which doesn't need to be included. Could you use `this.PropertiesUsedAsSutParameter` instead? Sorry about that.

Reply:
Made changes to use  Using PropertiesUsedAsSutParameter not Properties